modules/artifact-activation/src/main/java/org/mule/runtime/module/artifact/activation/internal/classloader/AbstractArtifactClassLoaderConfigurationAssembler.java

```diff
@@ -72,6 +73,10 @@ public ClassLoaderModel createClassLoaderModel() {
                                                                   PRIVILEGED_EXPORTED_PACKAGES)),
                                        new HashSet<>(getAttribute(muleArtifactLoaderDescriptor.getAttributes(),
                                                                   PRIVILEGED_ARTIFACTS_IDS)));
+    } else if (isMuleFramework()) {
```

Comment:
The `muleArtifactLoaderDescriptor` might not be `null` and yet the `exportedPackages` attribute not be present, so I think this should be done in that case too.

Reply:
I think the idea is only to export everything If the classLoaderModelLoaderDescriptor is null like when the app only contains the minMuleVersion

Reply:
this should be some setting set specifically instead of asking whete is it being called from